.github/workflows/ci.yaml

```diff
@@ -111,7 +111,7 @@ jobs:
           grype dir:. --exclude ./hack
         echo "##[endgroup]"
         echo "##[group]Scan manager image"
-          grype registry:registry.local:5000/service-bindings/manager:latest
+          grype registry.local:5000/service-bindings/manager:latest
```

Comment:
I tend to think the previous value was correct. While this seems to work, being explicit about the location of the artifact to scan is better.